onset_maproom/calc.py

```diff
@@ -18,13 +34,38 @@ def water_balance(
     sminit,
     time_coord="T",
 ):
-    """Estimates soil moisture from
-    rainfall,
-    evapotranspiration,
-    total available water and
-    intial soil moisture value, knowing that:
-    sm(t) = sm(t-1) + rain(t) - et(t)
-    with roof and floor respectively at taw and 0 at each time step.
+    """ Calculate soil moisture.
+
+        Estimates soil moisture from:
+            Rainfall, evapotranspiration, total available water and intial soil moisture value. 
+        Knowing that:
+            sm(t) = sm(t-1) + rain(t) - et(t)
```

Comment:
use same terms as the function's parameters (so here daily_rain rather than rain). and put them inside `` (see numpy doc doc for more on this).

Reply:
not seeing something regarding using '' - is this to show code / equations like we do `here on github`

I tried playing around with this but it did not work the same way when I render the html.

Reply:
Sorry... I am not being very clear, am I? :)
I refer to the ` symbol used to quote parameters in all parts of the text. See for instance `x` in the guide [Parameters](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) section.

Or see what I did in this on-going [PR](https://github.com/iridl/python-maprooms/pull/82/files).